WatchDogIntelliJPlugin/WatchDog/.idea/compiler.xml

```diff
@@ -18,6 +18,21 @@
       <profile default="true" name="Default" enabled="false">
```

Comment:
Yes: https://intellij-support.jetbrains.com/hc/en-us/articles/206827587-How-to-manage-projects-under-Version-Control-Systems (link that you sent me :) )